runtime/doc/options.txt

```diff
@@ -4872,8 +4741,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	languages, no matter what you set 'mkspellmem' to.
 
 				   *'modeline'* *'ml'* *'nomodeline'* *'noml'*
-'modeline' 'ml'		boolean	(Vim default: on (off for root),
-				 Vi default: off)
+'modeline' 'ml'		boolean	(default: on (off for root))
```

Comment:
Note that removing support for `'compatible'` was a different case. It will allow us to remove implementations of some Vi behaviors that no one uses. But setting an *option* (`set foo&vi`) does not require any special "implementation".